frontend/src/views/Home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from '../components/Button/Button';
+
+const Home = () => (
+  <div className="Home">
+    <div className="Home-hero">
+      <h1 className="Home-heading">Two line description<br />of the idea and service.</h1>
```

Comment:
This should be styled by the `h1` style, making the class unnecessary. Otherwise we end up repeating styles

Reply:
Yep! 100%